tests/functional/testplan/runnable/interactive/test_api.py

```diff
@@ -424,7 +433,7 @@ def test_run_suite(plan):
 
     timing.wait(
         functools.partial(
-            _check_test_status, suite_url, "failed", updated_json["hash"]
+            _check_runtime_status, suite_url, "finished", updated_json["hash"]
```

Comment:
I don't think we need `_check_test_status()` function anymore, `_check_runtime_status()` basically just replaces it.

Test logic should be:

1. Trigger test(s) to run
2. Poll for "runtime_status" to be FINISHED
3. Check that "status" is PASSED or FAILED as expected

Reply:
yea, i agree.